product-development/README.md

```diff
@@ -40,6 +40,7 @@ When picking a high priority unstarted epic, look at the epics _above the line_
 - My skill set is better suited for the third highest priority epic, so I'll pick that one, and save the first two for other folks who can finish them more quickly.
 - The fourth highest priority epic looks to be small so I'll pick that one first and finish it as a quick win before the end of the week.
 - My teammate has expertise in the highest priority epic and I'd like to collaborate and learn from them. But they aren't free just yet. I'll work on the third highest priority epic for now and plan to come back to pair with my teammate next week.
+- I see a few folks already working on Tilt Cloud epics, so I'll pick a Tilt epic to work on to try to load balance product iterations.
```

Comment:
As of right now, with the exterminator rotation, we only have 4 full time engineers at most, sometimes only 3. Is it appropriate to state that as a goal we want people working on a diverse set of things when we also say that people should be working in **at least** pairs?

I'll note that this week everyone is working completely in silos as far as I am aware, with the possible exception of Han's work on the sample project with L.

I would rather have all 3/4 people working on the most important thing than have one person off doing less important work just to load balance product iterations. Especially in weeks where there are only 3 engineers around.

Reply:
> I would rather have all 3/4 people working on the most important thing than have one person off doing less important work just to load balance product iterations. Especially in weeks where there are only 3 engineers around.

Thanks @jazzdan.  Agree we can’t really load balance and collaborate if we have so little people. Happy to close this PR. (After we’ve gotten some more feedback from Maia who brought up the topic.)

With Han close to wrapping up the sign in work (as they mentioned in the epics meeting), does it make sense for you and Matt to work on something together? I think this makes sense what you said regarding more collaboration and would love to see us driving toward that. And I like how you broke down the epics you’ve been working on and I’m guessing multiple people could work on it together?

Reply:
Well there's still more work to do on this sign in stuff, I think it make more sense for @hyu and I to work together for the remainder of the week. Though that discussion probably doesn't belong here :)

Reply:
Thanks for the update @jazzdan